include/inicpp.h

```diff
@@ -581,6 +582,14 @@ namespace ini
             multiLineValues_ = enable;
         }
 
+        /** Sets whether or not to throw an exception if duplicate fields are found in a section.
+          * Default is false.
+          * @param enable enable or disable? */
+        void setExceptionWhenDuplicateField(bool enable)
```

Comment:
Rename the function to something like `allowOverwriteDuplicateFields`. Also Update the docs accordingly, i.e. it should say that if overwriting duplicate fields is not allowed, then the parser will throw an exception.